pkg/bastion/shell.go

```diff
@@ -1581,6 +1581,59 @@ GLOBAL OPTIONS:
 			Usage: "Manages users",
 			Subcommands: []cli.Command{
 				{
+					Name:        "create",
+					ArgsUsage:   "<email>",
+					Usage:       "Creates a new user",
+					Description: "$> user create bob@example.com\n   $> user create --name=Robert bob@example.com",
+					Flags: []cli.Flag{
+						cli.StringFlag{Name: "name", Usage: "Assigns a name to the user"},
+						cli.StringFlag{Name: "comment", Usage: "Adds a comment"},
+						cli.StringSliceFlag{Name: "group, g", Usage: "Names or IDs of `USERGROUPS` (default: \"default\")"},
+					},
+					Action: func(c *cli.Context) error {
+						if c.NArg() != 1 {
+							return cli.ShowSubcommandHelp(c)
+						}
+
+						if err := myself.CheckRoles([]string{"admin"}); err != nil {
+							return err
```

Comment:
@jwessel I think we should wrap errors with some more context
```go
if err != nil {
return fmt.Errorf("check roles error: %w, err)
} 
```

Reply:
Your suggestion is good, however, @jwessel is only following the current project coding style, so I prefer to focus on what brings the PR and have a dedicated "good practice" issue/PR on the side